Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -283,6 +290,16 @@ void CodeViewWidget::Update(const Core::CPUThreadGuard* guard)
   if (m_updating)
     return;
 
+  if (static_cast<DSP::LLE::DSPLLE*>(m_system.GetDSP().GetDSPEmulator()) != nullptr)
+  {
+    m_debug_interface =
+        &static_cast<DSP::LLE::DSPLLE*>(m_system.GetDSP().GetDSPEmulator())->m_debug_interface;
+  }
+  else
+  {
+    m_debug_interface = &m_system.GetPowerPC().GetDebugInterface();
+  }
```

Comment:
This code (in `CodeViewWidget::Update`) is probably the biggest problem with this branch as it currently stands; at least when I first wrote this the powerpc debug interface was a global variable so CodeViewWidget could statically reference it, but the DSP one wasn't and thus didn't exist e.g. before emulation started. (This may have changed when it was moved to `System`; I resolved merge conflicts a while back but didn't investigate it in detail.)

Probably the code view widget (and all of the debugger widgets) should instead have the debug interface passed as a parameter on startup, but that'd mean re-creating them every time the game is started. So I'm not sure what to do about that.